wandb/sdk/wandb_run.py

```diff
@@ -585,6 +621,10 @@ def __setstate__(self, state: Any) -> None:
         if not _attach_id:
             return
 
+        if state["_init_pid"] == os.getpid():
+            raise RuntimeError("attach in the same process is not supported")
+
+        self._is_attaching = ""
```

Comment:
wonder why this isnt covered?

Reply:
hmmm... does it counts the yea test for coverage since I only have a failure test in pytest

Reply:
i was trying to highlight and ask about the _is_attatching line which should always be executed for spawn... 

but now that i think about it, it is probably just python having non exact line counts.  if we add python 3.10 to yea, this will get covered (im guessing)

ignore this comment, it was just a curiosity.

Reply:
nope i agree it is strange :)